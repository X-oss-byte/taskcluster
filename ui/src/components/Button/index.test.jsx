import React from 'react';
import { render } from '@testing-library/react';
import Button from './index';
import { AuthContext } from '../../utils/Auth';

it('should render Button', () => {
  const { asFragment } = render(
    <AuthContext.Provider value={{ user: { id: 'userId' } }}>
      <Button className="className" id="id" onClick={jest.fn()}>
        Icon
      </Button>
    </AuthContext.Provider>
  );

  expect(asFragment()).toMatchSnapshot();
});
