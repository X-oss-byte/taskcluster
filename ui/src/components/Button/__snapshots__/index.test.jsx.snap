// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Button 1`] = `
<DocumentFragment>
  <button
    class="MuiButtonBase-root MuiButton-root MuiButton-text className"
    id="id"
    tabindex="0"
    type="button"
  >
    <span
      class="MuiButton-label"
    >
      Icon
    </span>
    <span
      class="MuiTouchRipple-root"
    />
  </button>
</DocumentFragment>
`;
